nautobot/extras/models/jobs.py

```diff
@@ -784,7 +809,7 @@ def log(
             log.save(using=JOB_LOGS)
 
         if logger:
-            if level_choice == LogLevelChoices.LOG_FAILURE:
+            if level_choice == LogLevelChoices.LOG_ERROR:
                 log_level = logging.ERROR
             elif level_choice == LogLevelChoices.LOG_WARNING:
                 log_level = logging.WARNING
```

Comment:
It could change to 

```py
log_method = getattr(logger, level_choice, None)
if log_method is not None and callable(log_method):
    log_method(message)
else:
    raise Exception(f"this should never happen")
```